webapp/src/js/controllers/inbox.js

```diff
@@ -217,21 +217,6 @@ const LAST_REPLICATED_SEQ_KEY = require('../bootstrapper/purger').LAST_REPLICATE
       }
     });
 
-    $scope.clearSelection = function() {
```

Comment:
This function was updating the `selected` property in the `InboxCtrl` scope (which is used as parent scope in LiveList, which is why all reports were highlighted), instead of the child scopes of `ContactsCtrl` and `ReportsCtrl`.